reference/5.1/Microsoft.PowerShell.Core/About/about_PSItem.md

```diff
@@ -0,0 +1,230 @@
+1, 2, 3 | Where-Object -FilterScript { ($PSItem % 2) -eq $true }
+```
+
+```output
+2
+```
+
+The **FilterScript** checks to see if the current object is even and filters
+out any odd values, returning only `2` from the original list.
+
+## ForEach and Where methods
+
+Both the [ForEach][04] and [Where][05] intrinsic methods for arrays take a
+scriptblock as an input parameter. You can use the `$PSItem` in those
+scriptblocks to access the current object.
+
+```powershell
+@(1, 2, 3).ForEach({ $PSItem + 1 }).Where({ ($PSItem % 2) -eq $true })
+```
+
+```output
+3
+```
+
+The scriptblock of the **ForEach** method increments the value of the current
+object by one. Then the scriptblock of the **Where** method checks whether the
+current object is even and filters out any odd numbers.
+
+## Delay-bind scriptblock parameters
+
+[Delay-bind scriptblocks][06] let you use `$PSItem` to define parameters for a
+pipelined cmdlet before executing it.
+
+```powershell
+dir config.log | Rename-Item -NewName { "old_$($_.Name)" }
+```
+
+## Switch statement scriptblocks
+
+In [switch statements][07], you can use `$PSItem` in both action scriptblocks
+and statement condition scriptblocks.
+
+```powershell
+$numbers = 1, 2, 3
+
+switch ($numbers) {
+    { ($PSItem % 2) -eq 0 } { "$PSItem is even" }
+    default { "$PSItem is odd" }
+}
+```
+
+```output
+1 is odd
+2 is even
+3 is odd
+```
+
+The statement condition scriptblock checks whether the current object is even.
+If it is, the associated action scriptblock outputs a message indicating the
+current object is even.
+
+The action scriptblock for the `default` condition outputs a message indicating
+the current object is odd.
+
+## Function process blocks
+
+When you define a [function][08], you can use `$PSItem` in the `process` block
+definition but not in the `begin` or `end` block definitions. If you
+reference `$PSItem` in the `begin` or `end` blocks, the value is `$null`
+because those blocks don't operate on each object in the pipeline.
+
+When you use `$PSItem` in the `process` block definition, the value is the
+value is the current object if the function is called in the pipeline and
+otherwise `$null`.
+
+```powershell
+function Add-One {
+    process { $PSItem + 1 }
+}
+
+1, 2, 3 | Add-One
+```
+
+```output
+2
+3
+4
+```
+
+> [!TIP]
+> While you can use `$PSItem` in [advanced functions][09], there's little
+> reason to do so. If your function is intended to input from the pipeline,
+> it's best to define parameters with one of the `ValueFromPipeline*` arguments
+> for the [Parameter][10] attribute.
+>
+> Using that functionality and the parameter binding for advanced functions
+> makes the implementation more explicit and predictable than processing the
+> current object to get the required values.
+>
+> One good use of `$PSItem` in advanced functions is to inspect the current
+> object itself for debugging or logging when the function has multiple
+> parameters that take input from the pipeline.
```

Comment:
Should we show an example?

Reply:
Added an example. I don't _love_ it, but I think it's clear.